apisix/cli/ops.lua

```diff
@@ -307,6 +307,10 @@ local function init(env)
               .. 'other than /root.')
     end
 
+    if env.ulimit <= 1024 then
+        print(str_format("Warning! Current user limits [%d] too small, please modify user limits by execute \'ulimt -n <new user limits>\' , otherwise the performance is low.", env.ulimit))
```

Comment:
```suggestion
        print(str_format("Warning! Current user limits [%d] is too small, please modify user limits by execute \'ulimt -n <new user limits>\' , otherwise the performance is low.", env.ulimit))
```

Reply:
Got it.